config/jobs/kubernetes/kops/kops-periodics-conformance.yaml

```diff
@@ -362,8 +362,8 @@ periodics:
           -v 2 \
           --up --down \
           --cloud-provider=aws \
-          --create-args="--image='099720109477/ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-arm64-server-20240701' --channel=alpha --networking=calico --zones=eu-central-1a --node-size=t4g.large --master-size=t4g.large --discovery-store=s3://k8s-kops-prow/discovery" \
-          --kops-version-marker=https://storage.googleapis.com/kops-ci/markers/release-1.27/latest-ci-updown-green.txt \
+          --create-args="--image='099720109477/ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-arm64-server-20240720' --channel=alpha --networking=calico --zones=eu-central-1a --node-size=t4g.large --master-size=t4g.large --discovery-store=s3://k8s-kops-prow/discovery" \
+          --kops-version-marker=https://storage.googleapis.com/k8s-staging-kops/kops/releases/markers/release-1.27/latest-ci.txt \
           --kubernetes-version=https://dl.k8s.io/release/stable-1.27.txt \
```

Comment:
@upodroid this new marker doesn't exist:

```
❯ curl https://storage.googleapis.com/kops-ci/markers/release-1.27/latest-ci-updown-green.txt
https://storage.googleapis.com/k8s-staging-kops/kops/releases/1.27.4+v1.27.2-7-g6552c301e8

❯ curl https://storage.googleapis.com/k8s-staging-kops/kops/releases/markers/release-1.27/latest-ci.txt
<?xml version='1.0' encoding='UTF-8'?><Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Details>No such object: k8s-staging-kops/kops/releases/markers/release-1.27/latest-ci.txt</Details></Error>
```


We may need a trivial commit merged to release-1.27

Reply:
Sorry, I missed that. Please tag me on the PR to +1 it